.eslintrc.js

```diff
@@ -77,6 +77,12 @@ module.exports = {
         "@typescript-eslint/no-shadow": ["error", { hoist: "all" }],
         "@typescript-eslint/no-explicit-any": "off", // Temporary
         "@typescript-eslint/array-type": "error",
+        "no-unused-expressions": "off",
```

Comment:
Since we extend [@typescript-eslint/recommended](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/src/configs/recommended.ts), this should be turned off automatically.

Reply:
`@typescript-eslint` recomends disabling the "native" rule 
https://typescript-eslint.io/rules/no-unused-expressions/#how-to-use
But the link you provided proves otherwise, so i'll remove it 👍